auth0/src/main/java/com/auth0/android/authentication/ParameterBuilder.kt

```diff
@@ -181,6 +181,12 @@ public class ParameterBuilder private constructor(parameters: Map<String, String
                 .setScope(OidcUtils.DEFAULT_SCOPE)
         }
 
+        @JvmStatic
+        public fun newBuilderWithRequiredScope(): ParameterBuilder {
+            return newBuilder()
+                .setScope(OidcUtils.REQUIRED_SCOPE)
```

Comment:
If the developer is already adding the `openid` scope, will this result in it being duplicated?

Reply:
If you see the `setScope` method, it will use `includeRequiredScope` which avoids repetition as you mentioned. There are existing internal mechanisms to avoid this.